react/features/participants-pane/components/native/ContextMenuMore.js

```diff
@@ -9,22 +9,22 @@ import { useDispatch, useSelector } from 'react-redux';
 import { openDialog, hideDialog } from '../../../base/dialog/actions';
 import BottomSheet from '../../../base/dialog/components/native/BottomSheet';
 import {
-    Icon, IconMicDisabledHollow,
+    Icon,
     IconVideoOff
 } from '../../../base/icons';
 import {
     getLocalParticipant,
     getParticipantCount
 } from '../../../base/participants';
-import { BlockAudioVideoDialog } from '../../../video-menu';
 import MuteEveryonesVideoDialog
     from '../../../video-menu/components/native/MuteEveryonesVideoDialog';
 
 import styles from './styles';
 
 export const ContextMenuMore = () => {
     const dispatch = useDispatch();
-    const blockAudioVideo = useCallback(() => dispatch(openDialog(BlockAudioVideoDialog)), [ dispatch ]);
+
+    // const blockAudioVideo = useCallback(() => dispatch(openDialog(BlockAudioVideoDialog)), [ dispatch ]);
```

Comment:
❗

Reply:
Ah this is for the avmoderation stuff, right? Can you mention that in a comment?

Reply:
Removed it.